hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -40,32 +140,48 @@ def perform_enque_for_review(match_message: MatchMessage):
     logger.debug("enqued for review")
 
 
-possible_actions = {
-    "WRITEBACK_IN_REVIEW": perform_writeback_in_review,
-    "ENQUE_FOR_REVIEW": perform_enque_for_review,
-}
+class LabelWithConstraints(Label):
```

Comment:
Nice

Reply:
Thx! :) My first stab at something with inheritance in Python!